script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -152,9 +171,10 @@ Even though I knew that people gathered here like this, I'd never experienced it
 [sha:b2478d18b719c22dc758998adf58c4ebf2bbd585]{
 -- Page 307, Offset 25033.
 -- 情報だけの世界はそれはそれで<真|しん>だけど、受肉からの感覚は正しい理論より芯がある。
-// Sorry, what?
-// I'm not sure how to work in the しん pun here...
-A world of only information would itself be 'truth'. However, the sensation stemming from incarnation is a core deeper than any correct theory.
+// Kinda slid around which word triggers the しん/しん, not sure I love it still
+A world built only of information is 'real' in and of itself,
```

Comment:
I think this is a pretty good solution. Well, it doesn't really have the same match but I think it's good